src/adapters/pulsepointLite.js

```diff
@@ -0,0 +1,89 @@
+import {createBid} from 'src/bidfactory';
+import {addBidResponse} from 'src/bidmanager';
+import {logError} from 'src/utils';
+import {ajax} from 'src/ajax';
+import {STATUS} from 'src/constants';
+
+function PulsePointLiteAdapter() {
+
+  var bidUrl = window.location.protocol + '//bid.contextweb.com/header/tag?';
+  var ajaxOptions = {
+    method: 'GET',
+    withCredentials: true,
+    contentType: 'text/plain'
+  };
+
+  function _callBids(bidderRequest) {
+    bidderRequest.bids.forEach(bidRequest => {
+      try {
+        var params = Object.assign({}, environment(), bidRequest.params);
+        var url = bidUrl + Object.keys(params).map(k => k + '=' + encodeURIComponent(params[k])).join('&');
+        ajax(url, (bidResponse) => {
+          bidResponseAvailable(bidRequest, bidResponse);
+        }, null, ajaxOptions);
+      } catch(e) {
+        //register passback on any exceptions while attempting to fetch response.
+        logError('pulsepoint.requestBid', 'ERROR', e);
+        bidResponseAvailable(bidRequest);
+      }
+    });
+  }
+
+  function environment() {
+    var pg = pageUrl();
+    return {
+      cn: 1,
+      ca: 'BID',
+      tl: 1,
+      'if': 0,
+      cwu: pg.pg,
+      cwr: pg.ref,
+      dw: document.documentElement.clientWidth,
+      cxy: document.documentElement.clientWidth + ',' + document.documentElement.clientHeight,
+      tz: new Date().getTimezoneOffset(),
+      ln: (navigator.language || navigator.browserLanguage || navigator.userLanguage || navigator.systemLanguage)
+    };
+  }
+
+  function pageUrl() {
```

Comment:
I would recommend using these function utils.getTopWindowUrl() and utils.getTopWindowLocation()